RFCs/rfc-0010-zk-passport.md

```diff
@@ -0,0 +1,147 @@
+## RFP: ZKPassport
```

Comment:
Can you migrate over relevant content from the prior RFC? I think there's a lot of relevant information in there about requirements, etc https://github.com/MinaFoundation/Core-Grants/pull/11